lib/screens/events.dart

```diff
@@ -53,16 +53,30 @@ class _EventsPageState extends State<EventsPage> {
       appBar: AppBar(
         title: Text('Events List'),
         actions: <Widget>[
-          Padding(
-            padding: EdgeInsets.all(8.0),
-            child: FlatButton(
-              onPressed: () {
-                AuthManager().signOut();
-                // Doing Pop and Push for the smooth closing animation
-                Navigator.of(context).pushReplacementNamed('/login');
-              },
-              child: Text('Logout'),
-            ),
+          Row(
+            children: <Widget>[
+              GestureDetector(
+                child: CircleAvatar(
+                  backgroundImage: CachedNetworkImageProvider(curUser.imgURL),
+                ),
+                onTap: () {
+                  // TODO: Implement Hero animation to go to Profile page
+                  print('Go to Profile');
```

Comment:
I also thought of the same, but what good will the profile page do?

Reply:
It's like Dashboard on Website.